server/shared/store/providers/local.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const isNil = require('lodash/isNil');
+const LRU = require('lru-cache');
+const micromatch = require('micromatch');
+const yup = require('yup');
+
+const schema = yup.object().shape({
+  maxAge: yup.number()
+});
+
+class Local {
+  constructor(config) {
+    config = schema.validateSync(config, { stripUnknown: true });
+    this.name = 'local';
+    this.maxAge = config.maxAge;
+    this.client = new LRU({ maxAge: config.maxAge });
+  }
+
+  set(key, value, ttl) {
```

Comment:
Would this work
```suggestion
  set(key, value, ttl = this.maxAge) {
```
?